client/app/error/error.html

```diff
@@ -0,0 +1,20 @@
+<div class="oui-message oui-message_error oui-message_l" 
```

Comment:
`oui-message` wouldn't work ?

Reply:
Nope, it won't work as I expect.
```<oui-message>``` will only display a block with the message.
What I want here is an entire page representing an error. See: https://www.ovh.com/manager/dedicated/index.html#/error for an example.